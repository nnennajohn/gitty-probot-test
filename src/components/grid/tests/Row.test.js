// @flow
import React from 'react';
import renderer from 'react-test-renderer';

import { Row } from '../index';

test('Styles matches Scrollable', () => {
  const tree = renderer.create(<Row />).toJSON();
  expect(tree).toMatchSnapshot();
  // expect(tree).toHaveStyleRule('display', 'flex');
  // expect(tree).toHaveStyleRule('flex', '0 1 auto');
  // expect(tree).toHaveStyleRule('flex-direction', 'row');
  // expect(tree).toHaveStyleRule('flex-wrap', 'wrap');
  // expect(tree).toHaveStyleRule('list-style', 'none');
});
