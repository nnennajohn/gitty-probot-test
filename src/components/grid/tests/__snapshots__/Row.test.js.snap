// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Styles matches Scrollable 1`] = `
.c0 {
  box-sizing: border-box;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex: 0 1 auto;
  -ms-flex: 0 1 auto;
  flex: 0 1 auto;
  -webkit-flex-direction: row;
  -ms-flex-direction: row;
  flex-direction: row;
  -webkit-flex-wrap: wrap;
  -ms-flex-wrap: wrap;
  flex-wrap: wrap;
  list-style: none;
  margin-right: -0.5rem;
  margin-left: -0.5rem;
}

@media only screen and (min-width:40em) {
  .c0 {
    margin-left: 1rem;
    margin-right: 1rem;
  }
}

<div
  className="c0"
/>
`;
